proactive-healthcheck/api-health-check.sh

```diff
@@ -0,0 +1,604 @@
+#!/bin/bash
+#set -x
+
+#input parameters
+#username to connect to the ProActive Portals
+username=changeme
+#password of the user connecting to the ProActive Portals
+proactiveAdminPasswordDecrypted=changeme
+#server DNS name or public IP
+serverDNSName=try.activeeon.com
+#server Port
+webPort=8443
+#web protocol, http or https
+webProtocol=https
+#name of one of the node sources, needed in a test to get its description
+nsName=local
+#timeout in seconds to check free nodes, job submission, and job status
+timeoutInSeconds=120
```

Comment:
for later versions, this can be defined as a flag such that the user can pass it to script.

Reply:
ok, it could be, but for now, all config options are defined in the script